analyzer/metadata-plugin/src/main/java/eu/fasten/analyzer/metadataplugin/MetadataDBExtension.java

```diff
@@ -166,6 +174,14 @@ public void consume(String record) {
                     }
                 });
             } catch (Exception expected) {
+                if (expected instanceof DataAccessException) {
```

Comment:
See below... better crate a second catch block.

Reply:
Addressed in [6948de1](https://github.com/fasten-project/fasten/pull/466/commits/6948de169066492fdb8a24447d2a0d96d4af31cc)